Docs/Book/Cookbook.md

```diff
@@ -267,7 +267,7 @@ Examples of using it:
     rms = rdMolAlign.AlignMol(mol1, mol2)
     tani = rdShapeHelpers.ShapeTanimotoDist(mol1, mol2)
     prtr = rdShapeHelpers.ShapeProtrudeDist(mol1, mol2)
-    print rms, tani, prtr
+    print('{0:.11f} {1:.11f} {2:.13f}'.format(rms, tani, prtr))
```

Comment:
I appreciate the effort, but I think it's distracting. We aren't testing the cookbook, so this degree of precision isn't necessary.
How about just doing a normal print here (and elsewhere)?